boards/esp32-ethernet-kit-v1_0/Makefile.features

```diff
@@ -0,0 +1,17 @@
+CPU_MODEL = esp32-wrover
+
+# common board and CPU features
+include $(RIOTBOARD)/common/esp32/Makefile.features
+
+# additional features provided by the board
+FEATURES_PROVIDED += periph_adc
```

Comment:
```suggestion
FEATURES_PROVIDED += esp_eth
FEATURES_PROVIDED += periph_adc
```

Reply:
I used periph_eth here instead, and esp_eth is added from the Makefile.dep (same as for esp32-olimex-evb)